agent/tcs/client/client.go

```diff
@@ -42,7 +43,9 @@ const (
 type clientServer struct {
 	statsEngine            stats.Engine
 	publishTicker          *time.Ticker
-	endPublish             chan struct{}
+	publishHealthTicker    *time.Ticker
```

Comment:
What's the motivation for introducing a new ticker for health metrics?

Reply:
If we reuse the ticker, the interval of each metrics will mess and can be 2*interval = 40s.